frontends/emoji.go

```diff
@@ -94,19 +94,19 @@ func (c *emojiConfig) formatCond(cur []string, cond iface.Cond, current bool) (r
 }
 
 func (c *emojiConfig) printAstro(astro iface.Astro) {
-        // print sun astronomy data if present
+	// print sun astronomy data if present
 	if astro.Sunrise != astro.Sunset {
-	    // half the distance between sunrise and sunset
-	    noon_distance := time.Duration(int64(float32(astro.Sunset.UnixNano() - astro.Sunrise.UnixNano()) * 0.5))
-	    // time for solar noon
-	    noon := astro.Sunrise.Add(noon_distance)
+		// half the distance between sunrise and sunset
+		noon_distance := time.Duration(int64(float32(astro.Sunset.UnixNano()-astro.Sunrise.UnixNano()) * 0.5))
+		// time for solar noon
+		noon := astro.Sunrise.Add(noon_distance)
```

Comment:
These seem like non standard indents. Please don't reformat code in PRs unless you are changing the line. This makes the diff harder to read.

Reply:
Yeah I know, not the best practice. Sorry. But go extensions just format the whole file. I will remove it